tools/project_manager/page_compare.php

```diff
@@ -167,10 +167,10 @@ public function radio_string($value, $label)
         return "<input type='radio' name='page_set' value='$value'$checked>" . html_safe($label);
     }
 
-    public function has_project_started_round($round_id)
+    private function has_project_started_round($round)
```

Comment:
Ditto adding function type defs here to assist with this change.

Reply:
Done.